sdk/schema/credname/names_test.go

```diff
@@ -0,0 +1,54 @@
+package credname
+
+import (
+	"testing"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGettingCredentialIDsFromNames(t *testing.T) {
+	names := []sdk.CredentialName{
+		APIClientCredentials,
+		APIKey,
+		APIToken,
+		AccessKey,
+		AccessToken,
+		AppPassword,
+		AppToken,
+		AuthToken,
+		CLIToken,
+		Credential,
+		Credentials,
+		DatabaseCredentials,
+		LoginDetails,
+		PersonalAPIToken,
+		PersonalAccessToken,
+		RegistryCredentials,
+		SecretKey,
+	}
+
+	expectedIDs := []string{
+		"api_client_credentials",
+		"api_key",
+		"api_token",
+		"access_key",
+		"access_token",
+		"app_password",
+		"app_token",
+		"auth_token",
+		"cli_token",
+		"credential",
+		"credentials",
+		"database_credentials",
+		"login_details",
+		"personal_api_token",
+		"personal_access_token",
+		"registry_credentials",
+		"secret_key",
+	}
```

Comment:
Are we expecting contributors to keep this list up to date? If not, I'd recommend making this slightly less realistic-looking and more like a unit test that is also testing more edge cases.

Reply:
Added some additional edge case in addition to the list of the copy pasted existing credential names.